### PR TITLE
STL10 dataset

### PR DESCRIPTION
**Pull request by arthurdouillard**
- STL10

### CODE REVIEW

**Comment by arthurdouillard** [comment]
Merge #197 first

**Comment by TLESORT** [comment]
Also some conflict here

File: continuum/datasets/stl10.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+from torchvision import datasets as torchdata
+
+from continuum.datasets import _ContinuumDataset
+
+
+class STL10(_ContinuumDataset):
+    def __init__(
+            self, data_path: str = "", train: Union[bool, str] = True, download: bool = True):
+
+        super().__init__(data_path=data_path, train=train, download=download)
```
Comment (TLESORT) [inline continuum/datasets/stl10.py:13]: for code clarity tasktype can be mentionned here (even if it is the default one)

Comment (arthurdouillard) [reply continuum/datasets/stl10.py:13]: Done

**Pull request merged**